tests/integration/psync2-master-restart.tcl

```diff
@@ -120,10 +120,10 @@ start_server {} {
         $master debug set-active-expire 0
         for {set j 0} {$j < 1024} {incr j} {
             $master select [expr $j%16]
-            $master setex $j 1 somevalue
+            $master setex $j 10 somevalue
```

Comment:
i think you meant to change this one to `psetex` too.
btw, i think psetex and setex / setnx etc are meant to be deprecated.. 
i prefer to use SET PX in new code.

Reply:
oops, my fault, fixed